exporter/lambda/cross_region_import_replication.py

```diff
@@ -53,16 +56,12 @@ def _lambda_handler():
         "Resources": master_template_resources,
     }
 
-    cloudformation_client = boto3.client("cloudformation")
 
     _upload_template(os.environ["GENERATED_STACK_NAME"], json.dumps(master_template))
     template_url = _build_unsigned_url(os.environ["GENERATED_STACK_NAME"])
 
     try:
-        cloudformation_client.update_stack(
-            StackName=os.environ["GENERATED_STACK_NAME"],
-            TemplateURL=template_url,
-        )
+        _update_stack(template_url)
     except botocore.exceptions.ClientError as e:
```

Comment:
I think we should ensure that we do not raise if we reraise the error where the stack is in progress

Reply:
Like we said yesterday. You could just add an `elif "UPDATE_IN_PROGRESS`.